docs/reference/settings/security-settings.asciidoc

```diff
@@ -1675,18 +1675,19 @@ Provider. The value for this setting should be provided by your OpenID Connect P
 // tag::oidc-op-jwkset-path-tag[]
 `op.jwkset_path` {ess-icon}::
 (<<static-cluster-setting,Static>>)
-The path or URL to a JSON Web Key Set with the key material that the OpenID Connect
-Provider uses for signing tokens and claims responses.
-If a path is provided, then it is resolved relative to the {es} config
-directory.
-If a URL is provided, then it must be either a `file` URL or a `https` URL.
+
+The file name or URL to a JSON Web Key Set (JWKS) with the public key material that
+the OpenID Connect Provider uses for signing tokens and claims responses. A value is considered a file name
```

Comment:
Nit: It reads like idP signs with the public key. But it actually uses the private key associated with the public key. How about tweaking it a bit like the follows?
```suggestion
The file name or URL to a JSON Web Key Set (JWKS) with the public key material used to verify
tokens and claims responses signed by the OpenID Connect Provider. A value is considered a file name
```